thrift-parser.js

```diff
@@ -366,6 +410,34 @@ module.exports = (buffer, offset = 0) => {
     return result;
   };
 
+  const readUnion = () => {
+    let subject = readKeyword('union');
+    let name = readName();
+    let items = readUnionBlock();
```

Comment:
I think a parser should keep as much original information as possible, even though it's meaningless in some cases.